my-app/src/components/use-cases/useCases.css

```diff
@@ -0,0 +1,24 @@
+.use-cases {
+  height: 100%;
+  width: 15%;
+  background-color: #282828;
+  color: white;
+  float: left;
+  padding: 20px;
+  text-align: left;
+  font-weight: bold;
+}
+
+.title {
+  font-size: 20px;
+}
+
+.use-cases .use-case-item {
+  line-height: 22px;
+  font-size: 14px;
+  padding: 6px 0;
+}
+
+.green {
+  background-color: #09bba6 !important;
+}
```

Comment:
https://stackoverflow.com/questions/5116288/how-to-select-an-element-with-2-classes

remove `important`